x/interchainquery/types/query.go

```diff
@@ -1,9 +1,18 @@
 package types
 
-import sdk "github.com/cosmos/cosmos-sdk/types"
+import (
+	fmt "fmt"
+	time "time"
+)
 
 // Check if a query has timed-out by checking whether the block time is after
 // the timeout timestamp
-func (q Query) HasTimedOut(ctx sdk.Context) bool {
-	return q.TimeoutTimestamp < uint64(ctx.BlockTime().UnixNano())
+func (q Query) HasTimedOut(currentBlockTime time.Time) bool {
+	return q.TimeoutTimestamp < uint64(currentBlockTime.UnixNano())
+}
+
+// Prints an abbreviated query description for logging purposes
+func (q Query) Description() string {
+	return fmt.Sprintf("QueryId: %s, QueryType: %s, ConnectionId: %s, QueryRequest: %v",
```

Comment:
This makes a lot more sense to me to have this helper on the query.  I get that it makes the logging have to split up the string generation and log output steps but I still like this better personally.

Reply:
yeah I can't stand the long log statements